File: dev-docs/snippets/group-type-configs/get.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Fetch the configuration for one group type.
export async function get(client: JsBaoClient, groupType: string) {
  // #region example
  const config = await client.groupTypeConfigs.get(groupType);
  // #endregion example
  return config;
}

File: dev-docs/snippets/group-type-configs/get.swift
import JsBaoClient

// Fetch the configuration for one group type. Returns the typed
// `GroupTypeConfigInfo`. The `groupType` path segment is percent-encoded.
func get(client: JsBaoClient, groupType: String) async throws {
  // #region example
  let config = try await client.groupTypeConfigs.get(groupType: groupType)
  let autoAddCreator = config.autoAddCreator
  // #endregion example
  _ = (config, autoAddCreator)
}
